runtime/src/test_utils.rs

```diff
@@ -788,6 +810,12 @@ impl<BS: Blockstore> Runtime for MockRuntime<BS> {
         if let &Payload::ID(id) = address.payload() {
             return Some(id);
         }
+        if Protocol::Delegated == address.protocol() {
+            return self
+                .delegated_addresses_source
+                .get(address)
+                .and_then(|a| self.resolve_address(a));
+        }
```

Comment:
This change will disappear of the delegated addresses are mapped in the existing mapping.